.github/workflows/publish.yaml

```diff
@@ -0,0 +1,29 @@
+name: Publish
```

Comment:
Nit: We have many things to publish (source tarballs, packages), so I would name this workflow file `module-api.yaml` or `publish-module-api.yaml` (and the same in the `name` field inside the file).